charts/cloudzero-agent/README.md

```diff
@@ -17,255 +17,36 @@ For the latest release, see [Releases](https://github.com/Cloudzero/cloudzero-ch
-
-There are several options for provisioning this certificate. The default is to use a third party tool, [cert-manager](https://cert-manager.io/). If you would prefer not to use cert-manager, see the [Certificate Management](#certificate-management) section for other options.
-
-To use `cert-manager` for certificate management, first install the `cert-manager` CRDs with the following:
-```console
-helm install <RELEASE_NAME> cloudzero/cloudzero-agent \
-    --set insightsController.webhook.issuer.enabled=false \
-    --set insightsController.webhook.certificate.enabled=false \
-    --set insightsController.cert-manager.installCRDs=true
-```
-Alternatively, [install the cert-manager CRDs directly](https://cert-manager.io/docs/installation/helm/) with:
+#### 1. Create and Configure a Values File
 
-```console
-kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.16.2/cert-manager.yaml
-```
-
-3. Fill out all required fields in the `configuration.example.yaml` file in this directory. Rename the file as necessary. Below is an example of a completed configuration file:
 ```yaml
-# -- Account ID of the account the cluster is running in. This must be a string - even if it is a number in your system.
-cloudAccountId: YOUR_CLOUD_ACCOUNT_ID
-# -- Name of the clusters.
-clusterName: YOUR_CLUSTER_NAME
-# -- Region the cluster is running in.
-region: YOUR_CLOUD_REGION
-# -- CloudZero API key. Required if existingSecretName is null.
-apiKey: YOUR_CLOUDZERO_API_KEY
-# -- If set, the agent will use the API key in this Secret to authenticate with CloudZero.
-existingSecretName: YOUR_EXISTING_API_KEY_K8S_SECRET
-
-# label and annotation configuration (managed in the 'insightsController' section). See the below 'Labels and Annotations' section for more details.
-insightsController:
-  # -- By default, a ValidatingAdmissionWebhook will be deployed that records all created labels and annotations
-  enabled: true
-  labels:
-    # -- This value MUST be set to either true or false. The installation will fail otherwise
-    enabled: true
-    # -- This value MUST be set to a list of regular expressions which will be used to gather labels from pods, deployments, statefulsets, daemonsets, cronjobs, jobs, nodes, and namespaces
-    patterns:
-      - '^foo' # -- Match all labels whose key starts with "foo"
-      - 'bar$' # -- Match all labels whose key ends with "bar"
-  annotations:
-    # -- By default, the gathering of annotations is not enabled. To enable, set this field to true
-    enabled: false
-    patterns:
-      - '.*' # -- match all annotations. This is not recommended.
-cert-manager:
-  # -- Your cluster may already have cert-manager running, in which case this value can be set to false.
-  enabled: true
-```
-
-4. Install the helm chart using the completed configuration file:
-```console
-helm install <RELEASE_NAME> cloudzero/cloudzero-agent -f configuration.example.yaml
-```
-
-### Mandatory Values
```

Comment:
Let's move this to the values file and use that as the source of truth...also, let's adjust to where all but only the necessary have sensible defaults